### DISCUSSION TITLE
Bevy Log File Output

### DISCUSSION THREAD

**Discussion by samcarey** [bevyengine/bevy#5230] [Q&A] [answered]
Is there a way to send Bevy logs to a file? 
Preferably I'd like to print to both console and the file.

**Comment by bjorn3** [comment, marked as answer]
You can use your own tracing subscriber using `bevy::utils::tracing::subscriber::set_global_default()`. Make sure to disable the `LogPlugin` though as tracing only allows setting a global subscriber once. See https://github.com/bevyengine/bevy/blob/main/crates/bevy_log/src/lib.rs for the way the current console printing is implemented.

I do think we should make it easier to set up common log configurations like logging to files. Feel free to open an issue for that.

**Reply by samcarey** [reply]
OK thanks! I was able to get something working like you said, and I opened #5233.
